src/org/usfirst/frc4904/robot/RobotMap.java

```diff
@@ -82,6 +82,8 @@
 			public static final double TURN_EXP = 2;
 			public static final double DEFENSE_MANIPULATOR_SPEED_SCALE = 0.25;
 			public static final double OPERATOR_Y_OUTTAKE_UPPER_THRESHOLD = -0.5;
+			public static final double FLYWHEEL_SPINUP_AFTER_HOODUP_DELAY = 0.25;
+			public static final double HOODDOWN_AFTER_TRIGGERRELEASE_DELAY = 1;
```

Comment:
Can we have units for these two constants?
`SECONDS_HOODDOWN_AFTER_TRIGGRRELEASE_DELAY`, etc?